python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/search.py

```diff
@@ -365,3 +411,43 @@ def get_glossary_terms(self, count: int = 1000) -> SearchResponse:
         return SearchResponse(
             total_results=response["total"], page_results=page_results
         )
+
+    def _parse_container(self, entity: dict[str, Any], matches) -> SearchResult:
+        """
+        Map a Container entity to a SearchResult
+        """
+        tags = parse_tags(entity)
+        last_updated = parse_last_updated(entity)
+        properties, custom_properties = parse_properties(entity)
+        owner_email, owner_name = parse_owner(entity)
+
+        metadata = {
+            "owner": owner_name,
+            "owner_email": owner_email,
+            "entity_sub_type": (
+                entity.get("subTypes", {}).get("typeNames", ["Container"])
+                if entity.get("subTypes") is not None
+                else ["Container"]
+            ),
+        }
+
+        fqn = (
```

Comment:
I think this logic is repeated in a few places, so could extract a common function here as well